crates/rspack_plugin_runtime/src/runtime_module/css_loading.rs

```diff
@@ -7,21 +7,22 @@ use rspack_identifier::Identifier;
 use rustc_hash::FxHashSet as HashSet;
 
 use super::utils::chunk_has_css;
-use crate::runtime_module::{render_condition_map, stringify_chunks};
+use crate::{
+  get_chunk_runtime_requirements,
+  runtime_module::{render_condition_map, stringify_chunks},
+};
 
-#[derive(Debug, Default, Eq)]
+#[derive(Debug, Eq)]
 pub struct CssLoadingRuntimeModule {
   id: Identifier,
   chunk: Option<ChunkUkey>,
-  runtime_requirements: RuntimeGlobals,
```

Comment:
`runtime_requirements` should use `get_chunk_runtime_requirements`, this is a copied value, but actually we need a reference, but using reference will break the borrow checker rules, so using `get_chunk_runtime_requirements` to get it